packages/renderer/src/lib/onboarding/Onboarding.svelte

```diff
@@ -220,7 +220,9 @@ async function restartSetup() {
 </script>
 
 {#if activeStep}
-  <div class="flex flex-col bg-[#36373a] h-full">
+  <div
+    class="flex flex-col bg-[#36373a] h-full overflow-y-auto w-full overflow-x-hidden"
```

Comment:
it's not related to this PR but IMHO we should use a color from our palette colors

@mairin what is the color that we should use instead of `#36373a`

Reply:
charcoal-500 for sure!